python/src/aiconfig/editor/client/src/components/ConfigNameDescription.tsx

```diff
@@ -42,6 +36,11 @@ const useStyles = createStyles((theme) => ({
   },
 }));
 
+function isClickableChildElement(element: EventTarget | null) {
+  // Specifically handle markdown links and prism code copy svg elements
+  return element instanceof HTMLAnchorElement || element instanceof SVGElement;
```

Comment:
Why do we need to add this check when we didn't have it before?

Reply:
This is just checking if the target element passed in is either an anchor tag (link) or svg (what prism is using for the 'copy code' button)